e2e/provision/playbooks/roles/bootstrap/README.md

```diff
@@ -23,7 +23,7 @@ Available variables are listed below, along with default values (see defaults/ma
 | gtp5g_version                   | no       | v0.6.8                                                        |                           | GTP5G source code version                                                         |
 | gtp5g_tarball_url               | no       |                                                               |                           | GTP5G tarball URI                                                                 |
 | k8s.context                     | no       | kind-kind                                                     |                           | Kubernetes context to create gitea resources                                      |
-| k8s.version                     | no       | v1.27.1                                                       |                           | Kubernetes version used for the management cluster                                |
+| k8s.version                     | no       | v1.29.2                                                       |                           | Kubernetes version used for the management cluster                                |
```

Comment:
@electrocucaracha So we will have the same for management cluster and workload clusters right?

Reply:
No, in other to achieve that we need to modify the [workload cluster package](https://github.com/nephio-project/catalog/blob/main/infra/capi/cluster-capi-kind/cluster.yaml#L33)